### PR TITLE
SCS: update to version 3.2.7

### PR DESCRIPTION
**Pull request by kalmarek**

### CODE REVIEW

File: S/SCS/build_tarballs.jl [hunk 1]
```diff
@@ -45,3 +45,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat="1.6")
+
```
Comment (giordano) [inline S/SCS/build_tarballs.jl:48]:
```suggestion
```

File: S/SCS_GPU/build_tarballs.jl [hunk 2]
```diff
@@ -55,3 +55,4 @@ for platform in platforms
                    [dependencies; cuda_deps], julia_compat="1.6",
                    augment_platform_block=CUDA.augment)
 end
+
```
Comment (giordano) [inline S/SCS_GPU/build_tarballs.jl:58]:
```suggestion
```

File: S/SCS_MKL/build_tarballs.jl [hunk 3]
```diff
@@ -49,3 +49,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat="1.6")
+
```
Comment (giordano) [inline S/SCS_MKL/build_tarballs.jl:52]:
```suggestion
```

**Comment by kalmarek** [comment]
thanks @giordano!

**Pull request merged**